1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
```

Comment:
@tianon - Ignore the Linux binary bit. That's a client-only thing, not in containers or in server.